src/edu/csus/ecs/pc2/core/report/FileComparisonUtilities.java

```diff
@@ -0,0 +1,500 @@
+// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.report;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Level;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+import edu.csus.ecs.pc2.core.Constants;
+import edu.csus.ecs.pc2.core.StringUtilities;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.execute.ExecuteUtilities;
+import edu.csus.ecs.pc2.core.imports.clics.CLICSAward;
+import edu.csus.ecs.pc2.core.imports.clics.CLICSScoreboard;
+import edu.csus.ecs.pc2.core.imports.clics.FieldCompareRecord;
+import edu.csus.ecs.pc2.core.imports.clics.FileComparison;
+import edu.csus.ecs.pc2.core.log.StaticLog;
+import edu.csus.ecs.pc2.core.standings.json.TeamScoreRow;
+
+/**
+ * Utilities for file comparison
+ * 
+ * @author Douglas A. Lane, PC^2 team pc2@ecs.csus.edu
+ */
+public class FileComparisonUtilities {
+    
+    private static  ObjectMapper objectMapper = null;
+
+    public static FileComparison createScoreboardJSONFileComparison(String tsvFileName, String sourceDir, String targetDir, IFileComparisonKey fileComparisonKey) {
```

Comment:
The parameter name `tsvFileName` is _very_ misleading, since this method is _never_ called (and should never BE called) with a "TSV" file -- it is, and should only be -- called with a JSON file.

Reply:
parameter name updated.